src/checksum/zseries/crc32-s390x.c

```diff
@@ -78,7 +78,7 @@ DEFINE_CRC32_VX(__wt_crc32c_le_vx, __wt_crc32c_le_vgfm_16, __wt_crc32c_le)
 unsigned int
 __wt_cksum(const void *chunk, size_t len)
 {
-	return (~__wt_crc32c_le_vx(0xffffffff, chunk, len));
```

Comment:
@daveh86 Since we don't automatically build pull requests on zSeries, have you checked that you get the warning without this change and it goes away with the change?